src/test/java/org/apache/xml/security/test/dom/signature/ECDSASignatureTest.java

```diff
@@ -69,14 +70,14 @@ public ECDSASignatureTest() throws Exception {
 
         org.apache.xml.security.Init.init();
         try {
-            int javaVersion = Integer.parseInt(System.getProperty("java.specification.version"));
+            int javaVersion = Integer.getInteger("java.specification.version");
```

Comment:
This change is causing a NPE in the tests (Java 8):
```
[ERROR] org.apache.xml.security.test.dom.signature.ECDSASignatureTest.testKeyValue  Time elapsed: 0.001 s  <<< ERROR!
java.lang.NullPointerException
    at org.apache.xml.security.test.dom.signature.ECDSASignatureTest.<init>(ECDSASignatureTest.java:73)
```

Reply:
Argh, this was eclipse again, sorry.